blocks/hero-horizontal-tabs/hero-horizontal-tabs.js

```diff
@@ -128,7 +126,9 @@ export default function decorate(block) {
       button.classList.add('active');
       // add active class to parent li
       $tab.classList.add('active');
-      if (tab.$content) tab.$content.classList.remove('hidden');
+      if (tab.$content) {
```

Comment:
I realise that this may have been added by one of your previous commits, but I find it confusing that you have variables with the same name with/without a preceding `$` sign. Why have `tab` and `$tab`?  And why `$content`?

Reply:
+1 to this

Reply:
The `$content` I think I missed renaming at some point, so that's on me
As for the `$tab` thing, the dollar sign tab was used to represent the tab button and the no-dollar variable was for the "content", aka the section in the doc for that tab, they were both "tabs" in my mind, but I think "tabButton" would be 100% better instead of `$tab`